lib/sql_footprint/sql_statements.rb

```diff
@@ -3,12 +3,17 @@
 
 module SqlFootprint
   class SqlStatements
+    NEWLINE = "\n".freeze
     def initialize
       @statements = Set.new
     end
 
     delegate :to_a, :add, :sort, :count, to: :statements
 
+    def to_s
+      statements.sort.join(NEWLINE) + NEWLINE
```

Comment:
@brandonjoyce I think I would write a different serializer if the format was going to be different, right?

What about:
```ruby
FootprintSerializer.new(statements).to_s
```

btw I was originally suggesting the `to_file` method because i thought it would be nice to move the `write` method out of the `SqlCapture` class.